packages/hash/api/src/db/adapter.ts

```diff
@@ -545,11 +558,12 @@ export interface DbClient {
   /**
    * Create an aggregation for an entity.
    *
-   * @param params.sourceAccountId the account id of the source entity
-   * @param params.sourceEntityId the entity id of the source entity
-   * @param params.path the aggregation path
-   * @param params.operation the aggregation operation
-   * @param params.createdByAccountId the account id of the user that created the aggregation
+   * @param params.sourceAccountId - the account id of the source entity
+   * @param params.sourceEntityId - the entity id of the source entity
+   * @param params.path - the aggregation path
+   * @param params.operation - the aggregation operation
+   * @param params.createdByAccountId - the account id of the user that created the aggregation
```

Comment:
It could be confusing that this field 'disappears', i.e. it doesn't end up in the db - we could either rename the param `appliedToSourceByAccountId`, or note here that it 'becomes' `appliedToSourceByAccountId`

Reply:
I see what you're saying, I feel that for a `create` method a `createdBy` parameter most intuitively conveys that the parameter represents the user that is responsible for the call of this method. I think if this field were to be renamed to `appliedToSourceByAccountId`, I should also rename the method to `applyAggregationToEntity` and I'm not sure I prefer that to the more straightforward `createAggregation`. Would you prefer `applyAggregationToEntity`?

For now I've clarified in the documentation what the `createdAtByAccountId`/`deletedAtByAccountId` fields represents in https://github.com/hashintel/hash/pull/484/commits/7ef96fd5aa9766a6db237409f1b2b58b44010647

Reply:
Understood - clarification in the documentation is fine